src/template.js

```diff
@@ -0,0 +1,50 @@
+import { select } from 'd3-selection'
+import flamegraph from './flamegraph'
+import './flamegraph.css'
+import './template.css'
+
+class FlameGraphUI {
+    constructor (stacks, options) {
+        this.stacks = stacks || { name: 'placeholder', value: 0, children: [] }
```

Comment:
Might want to leave undefined and fail if stack is not defined.

Reply:
Fail with a JS error? I put the placeholder here so for ppl who don't know about the JS console etc. who open the empty template by mistake see something. I've updated the text now to `Please replace /** @flamegraph_params **/ with flame graph stacks in the template.`

Reply:
Thinking more of an error message saying stacks were not loaded.

Reply:
done